general_ledger/templates/detail_macro.html

```diff
@@ -8,14 +8,17 @@
     <th style="text-align: right;">{{ _('Balance') }}</th>
   </tr>
   {% for k, record in records.items() %}
-    {% if not record.party_required %}
-      <tr>
-        <td colspan="2" class="bold">{{ record.code }}</td>
-        <td class="bold">{{ record.account }}</td>
-        <td colspan="2" style="text-align: right;">{% if record.lines %}{{ _('Previous balance...') }}{% endif %}</td>
-        <td style="text-align: right;">{% if record.lines %}{{ record.previous_balance|render }}{% endif %}</td>
-      </tr>
+    {% if (not record.party and record.party_required) or (record.party and not record.party_required) %}
+      {% set style = 'color: #FF0000;' %}
+    {% else %}
+      {% set style = '' %}
```

Comment:
I don't think coloring this exception is appropriate. It is really a bug in the system so it makes no sense to add a feature to show the user a bug in the software.